shared/util.go

```diff
@@ -117,7 +117,7 @@ func ReadToJSON(r io.Reader, req interface{}) error {
 }
 
 func GenerateFingerprint(cert *x509.Certificate) string {
-	return fmt.Sprintf("% x", sha256.Sum256(cert.Raw))
+	return fmt.Sprintf("%x", sha256.Sum256(cert.Raw))
```

Comment:
It won't matter for the client as it doesn't cache fingerprints or expects them in a given format, but this will likely cause authentication failures due to a mismatch between the fingerprint in the database and that computed on connection.

So I think we need a DB update function which will reformat all the existing fingerprints.